orangecontrib/spectroscopy/widgets/owspectra.py

```diff
@@ -1009,6 +1067,71 @@ def grid_apply(self):
         self.plot.showGrid(self.show_grid, self.show_grid, alpha=0.3)
         self.show_grid_a.setChecked(self.show_grid)
 
+    def find_peak_variables(self):
+        if self.maximum_point is None:
+            self.maximum_point = np.amax(self.data.X[:, self.data_xsind])
+        if self.minimum_point is None:
+            self.minimum_point = np.amin(self.data.X[:, self.data_xsind])
+        if self.start_point is None:
+            self.start_point = round(np.median(getx(self.data)), 2)
```

Comment:
Do not round here. Only round strings at the output. Who knows what is the user's range (perhaps it can be 0.0000001 to 0.0000300).